guide/src/ref/packet-log.md

```diff
@@ -103,18 +230,164 @@ protocol "tcp.flow_id:32,tcp.seq:32,tcp.ack:32,ip.id:16,ip.frag:16,ip.checksum:1
+- `src.ip`, `dst.ip`, `src.port`, `dst.port`: The source IPv4 address, destination IPv4 address, source TCP port number, description TCP port number.
+- `Options on SYN/SYN_ACK`: As flows are uni-directional here, only one valid of SYN(TCP client side) or SYN_ACK (TCP server side) would exist in a TCP connection. The TCP options fileds in such pakcets are recorded in raw here. This field has a fixed length of 40 Bytes, and zero-padding at the end if the tcp options as less than 40 Bytes. 40 Bytes is the upper limit in TCP protocol: a TCP header can be at most 15 \* 4 = 60 Bytes in length, and the first 20 bytes are used for fixed fields in the TCP header.
+
+#### Chunk Prologue Entry
+
+A chunk prologue entry is a log entry with `LH.typ` = 15, which is used in raw-mode packet logs.
+
+```text
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|       LH.length       | LH.ty.|       CH.length       | CH.ty.|
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                          Log Version                          |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                          Data Length                          |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                          Chunk Length                         |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                                                               |
++                                                               +
+|                                                               |
++                         custom payload                        +
+|                                                               |
++                                                               +
+|                                                               |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+```
+
+```bash
+protocol "LH.length:12,LH.ty.:4,CH.length:12,CH.ty.:4,Log Version:32,Data Length:32,Chunk Length:32,custom payload:128"
+```
+
+Where `CH` is short for Chunk Prologue Header, and the meaning of each field is as follows:
+
+- `CH.length` denotes the length for the chunk prologue entry, and `CH.ty` is always 0 for now.
+- `Log Version` is a magic number, and it is `0x20251120` in this specification.
+- `Data Length` is the sum of the length of this Chunk Prologue Entry and all the following contents (raw General Packet Entries for now).
+- `Chunk Length` is the logical length of the current chunk, i.e., it is expected to either meet EOF or to find the next Chunk Prologue, `Chunk Length` bytes since the offset of this log entry.
+
+And the custom payload for the `CH.ty` = 0, which is the only defined variant in this specification, is:
```

Comment:
Should reorganize this sentence to make it fluent and direct to understand the meaning